RELEASE.md

```diff
@@ -1,3 +1,40 @@
+# Release 2.2.0
+
+The 2.2 minor series tracks TensorFlow 2.2.
+
+## Features
+
+- Profile plugin now should be pip installed.
```

Comment:
Could you please provide here the package name that users should
install, and maybe slightly expand the expository text to communicate
the benefits of this change? (The new version works in Chrome 80 and
Firefox, has better model insights, will be more actively maintained,
etc.)

Reply:
Clarified